### DISCUSSION TITLE
Carelink still not working, update done.

### DISCUSSION THREAD

**Discussion by JeandreBG** [NightscoutFoundation/xDrip#2916] [General]
I have done the latest update, but Xdrip is still not working. Restarted my phone too. 
![inbound6449586434092024430](https://github.com/NightscoutFoundation/xDrip/assets/103830279/87d5d1df-5540-4f3c-b048-6b9c649dc0e0)
Please assist. I'm attaching a screenshot of my system page. Thank you in advance.

**Comment by Navid200** [comment]
@benceszasz

**Comment by benceszasz** [comment]
@JeandreBG which country? Please paste a screenshot of the Error and Event list (you can find it in the three dots menu).

**Reply by JeandreBG** [reply]
I'm in South Africa. Logged out and back in on web and uploader with minimed mobile app.
![inbound2242922040895786152](https://github.com/NightscoutFoundation/xDrip/assets/103830279/2edf4d72-1caa-487b-beda-84900f253583)
Thanks.

**Reply by benceszasz** [reply]
You only have a few login related errors and all of them are caused by network errors (socket timeout, hostname resolve errors). I think your data is not updated in CareLink Cloud, some had to uninstall Medtronic app and reinstall it again to have data in CareLink Cloud. To make sure you can set a detailed logging for CareLink Follower: Menu > Settings > Less Common Settings > Extra Logging Settings > Extra tags for Logging - paste here the following:
CareLinkFollow:d,CareLinkFollowDL:d,CareLinkFollowDP:d

This will trigger debug logging for all three main modules of CareLink Follower and you will be able to see detailed steps of CareLink Follower in the Error and Event list.

**Reply by JeandreBG** [reply]
Thank you. I've done what you suggested. Still not working.
Please see below.
![inbound7112461829392004814](https://github.com/NightscoutFoundation/xDrip/assets/103830279/f6e2209d-c463-46da-b00f-22361c200506)

**Reply by benceszasz** [reply]
You can see in the logs, that login and data download works correctly. Based on the logs, however, the MiniMed mobile application is not connected to the pump: "Not connected to pump => ... ". Are you sure, that your MiniMed mobile application is connected to the pump? If it is connected are you sure, that data is uploaded to CareLink?
